charts/community/ai-lab/chatbot/OWNERS

```diff
@@ -0,0 +1,15 @@
+chart:
+  name: chatbot-ai-software-template
+  shortDescription: Chatbot AI Software Template
```

Comment:
See above

Reply:
Addressed in [4b440cf](https://github.com/openshift-helm-charts/charts/pull/1669/commits/4b440cf74bdb485cfffdca12d1358e72d661e29a)